<launch>
    <!-- include the official gazebo file -->

    <include file="$(find vrx_tasks)/launch/missioncontrol.launch">
        <arg name="world_name" value="navigation_task"/>
    </include>

    <!--include file="$(find vrx_navigation)/launch/lidar_node.launch">
</include-->

    <!--include file="$(find vrx_vision)/launch/object_server.launch">
        <arg name="use_camera" value="true"/>
    </include-->


</launch>
